scipy/optimize/_shgo.py

```diff
@@ -658,7 +658,7 @@ def _restrict_to_keys(dictionary, goodkeys):
     def init_options(self, options):
         """
         Initiates the options.
-        
```

Comment:
making white space changes in a PR that touches many files makes the PR hard to review.

Reply:
Sorry! My editor automatically removes trailing whitespace. This concerns 5 lines of code now. If this isn't acceptable, I'll be happy to re-add the whitespace.

Reply:
Re-rdded the whitespace now.